### PR TITLE
Update iberian_labs.md

### PR DESCRIPTION
**Pull request by joaovduarte**
Entry test

### CODE REVIEW

File: iberian_labs.md [hunk 1]
```diff
@@ -4,3 +4,17 @@ title: Iberian Labs
...
+University of Coimbra
+Coimbra, Portugal
+
+The **Proaction Lab** focuses on Cognitive Neuroscience approaches to unravel the processes associated with the perception and recognition of objects and actions, as well as how knowledge and information are stored and organized in the brain.
+
+Main research topics: *perception, objects, mind wandering, collaborative robots*
+
+Contact: Jorge Almeida, jorgealmeida@fpce.uc.pt
+
+Website: http://proactionlab.fpce.uc.pt/
```
Comment (eurunuela) [inline iberian_labs.md:18]:
```suggestion
Website: [http://proactionlab.fpce.uc.pt/](http://proactionlab.fpce.uc.pt/)
```

File: iberian_labs.md [hunk 2]
```diff
@@ -4,3 +4,17 @@ title: Iberian Labs
...
+
+Twitter: @proactionlab
```
Comment (eurunuela) [inline iberian_labs.md:20]:
```suggestion
Twitter: [@proactionlab](https://twitter.com/proactionlab)
```

File: iberian_labs.md [hunk 3]
```diff
@@ -4,3 +4,17 @@ title: Iberian Labs
 ---
 
 If you would like your research group to be listed in our website, please fill in <a href="https://forms.gle/X3XvHLDNbY7jF1QX6" target="_blank">this form</a>.
+
```
Comment (eurunuela) [inline iberian_labs.md:7]: Maybe we should add a title for the list of labs here.

File: iberian_labs.md [hunk 4]
```diff
@@ -4,3 +4,17 @@ title: Iberian Labs
 ---
 
 If you would like your research group to be listed in our website, please fill in <a href="https://forms.gle/X3XvHLDNbY7jF1QX6" target="_blank">this form</a>.
```
Comment (eurunuela) [inline iberian_labs.md:6]: We should mention that we will only show one lab per center.

**Review by eurunuela — CHANGES_REQUESTED** [review]
Hey @joaovduarte , thanks for opening this PR! First of many ;) 

I have tiny comments. Mainly, that we should make it easy for visitors to click on links.

File: iberian_labs.md (hunk 3 above)
Comment (joaovduarte) [reply iberian_labs.md:7]: Yes, good idea.

File: iberian_labs.md (hunk 4 above)
Comment (joaovduarte) [reply iberian_labs.md:6]: Yes, I will add this info.

**Pull request closed without merging**